scrapegraphai/graphs/base_graph.py

```diff
@@ -204,6 +210,8 @@ def _execute_standard(self, initial_state: dict) -> Tuple[dict, list]:
                 cb_total["successful_requests"] += cb_data["successful_requests"]
                 cb_total["total_cost_USD"] += cb_data["total_cost_USD"]
 
+            self.shared_state = state["shared_state"]
```

Comment:
- how many nodes change `shared_state` attributes requiring you to set it and re-set it in in the same traversal?